debian/clearwater-snmpd.postinst

```diff
@@ -58,6 +58,7 @@ case "$1" in
         cp /etc/snmp/snmpd.conf.clearwater-snmpd /etc/snmp/snmpd.conf
 
         # Create the folder to store the statistics UNIX domain sockets.
+        rm -rf /var/run/clearwater
```

Comment:
There were times when those domain IPs were leftover from a previous install and the C++ code was denied permission deleting the sockets, hence causing the SNMP stats and alarms to not work.

I could have made it more specific: rm -rf /var/run/clearwater/stats /var/run/clearwater/alarms